core/mpi/routes/paciente.ts

```diff
@@ -689,44 +673,33 @@ router.put('/pacientes/:id', (req, res, next) => {
                     }
                 }
                 // verifico si el paciente ya está en MPI
-                pacienteMpi.findById(query, (err3, patientFountMpi: any) => {
-                    if (err3) {
-                        return next(404);
-                    }
-                    if (patientFountMpi) {
-                        Auth.audit(newPatient, req);
-                    }
-                    newPatient.save((err2) => {
-                        if (err2) {
-                            return next(err2);
-                        }
-                        const nuevoPac = JSON.parse(JSON.stringify(newPatient));
-                        // delete nuevoPac._id;
-                        // delete nuevoPac.relaciones;
-
-                        const connElastic = new ElasticSync();
-                        connElastic.sync(newPatient).then(updated => {
-                            if (updated) {
-                                Logger.log(req, 'mpi', 'update', {
-                                    original: nuevoPac,
-                                    nuevo: newPatient
-                                });
-                            } else {
-                                Logger.log(req, 'mpi', 'insert', newPatient);
-                            }
-                            res.json(nuevoPac);
-                        }).catch(error => {
-                            return next(error);
-                        });
+                let patientFountMpi = await pacienteMpi.findById(query).exec();
```

Comment:
Se consulta si el paciente esta en MPI pero no se hace nada con ese dato.

Reply:
true that